server/suite_test.go

```diff
@@ -140,6 +140,7 @@ var beforeEach = func() {
 	Expect(err).To(BeNil())
 	serverConfig, err = config.DefaultConfig()
 	Expect(err).To(BeNil())
+	serverConfig.NRI.Enabled = true
```

Comment:
This is probably not a good idea at the moment. We should bump the NRI dependency to commit `807de1e0cffa` first (and adjust the related bits in the configuration). That eliminates the need for a global system-wide NRI configuration file, which was a bad idea to begin with.

@saschagrunert  @Tal-or I can prepare a PR for that. Then we could try on top of that this `serverConfig.NRI.Enabled = true` and see if we can iron out the any remaining related problems in the tests. WDYT ?

Reply:
agreed. I would say probably having dedicated tests with NRI enabled would be good

Reply:
Yes, the test failure is due to the missing configuration file indeed.
@haircommander I can address that, i'll edit the PR accordingly while we're waiting for the dependency bump.

Reply:
@haircommander, @Tal-or #6648 is the PR which bumps the NRI deps, eliminating the global config file (and it also adjusts the related configuration, documentation, etc. accordingly).

I'm see quite a bit of test failures there, but I see the same ones (BATS tests  42, 45, 46, 47) fail in many of the other recent PRs. Are these known to be broken now ?

Reply:
@Tal-or @hircommander I tested #6648 with 96cde59d99d8ce0974e6c6fbbefc60d176901471 cherry-picked to see how it goes. 3 of the unit test crashed with `internal/oci/Container` trying to dereference an unset nil `OCI Spec`, all others passed. [Added a fix for that crash](https://github.com/klihub/cri-o/tree/release-test) and am testing now to see if anything else comes up to fix. If things look fine, I'll update #6648 with that fix included.

Reply:
Thank you @klihub are the crashes in `internal/oci/Container` related to the one you mentioned here?
https://github.com/cri-o/cri-o/pull/6645#discussion_r1106421489

Reply:
@Tal-or Well, they are caused by an unset/`nil` OCI Spec being unconditionally dereferenced by `Spec()`. I've added [a fix](https://github.com/cri-o/cri-o/pull/6648/commits/bd7fb2416547dea477aa75de52e1f76082b8a0b3) for it in #6648.

@haircommander @Tal-or I've tested this rebased on #6648 and unit tests do pass now.